modules/api/pkg/handler/v2/routes_v2.go

```diff
@@ -8027,6 +8045,58 @@ func (r Routing) clusterBackupDownloadURL() http.Handler {
 	)
 }
 
+// Backup storage location.
+func (r Routing) createBackupStorageLocation() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+			middleware.SetClusterProvider(r.clusterProviderGetter, r.seedsGetter),
+			middleware.SetPrivilegedClusterProvider(r.clusterProviderGetter, r.seedsGetter))(backupstoragelocation.CreateEndpoint(r.userInfoGetter, r.backupStorageProvider, r.projectProvider, r.privilegedProjectProvider, r.settingsProvider)),
+		backupstoragelocation.DecodeCreateBSLReq,
+		handler.SetStatusCreatedHeader(handler.EncodeJSON),
+		r.defaultServerOptions()...,
+	)
+}
+
+func (r Routing) listBackupStorageLocation() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+			middleware.SetClusterProvider(r.clusterProviderGetter, r.seedsGetter),
+			middleware.SetPrivilegedClusterProvider(r.clusterProviderGetter, r.seedsGetter))(backupstoragelocation.ListEndpoint(r.userInfoGetter, r.projectProvider, r.privilegedProjectProvider, r.settingsProvider)),
+		backupstoragelocation.DecodeListBSLReq,
+		handler.SetStatusCreatedHeader(handler.EncodeJSON),
+		r.defaultServerOptions()...,
+	)
+}
+
+func (r Routing) getBackupStorageLocation() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+			middleware.SetClusterProvider(r.clusterProviderGetter, r.seedsGetter),
+			middleware.SetPrivilegedClusterProvider(r.clusterProviderGetter, r.seedsGetter))(backupstoragelocation.GetEndpoint(r.userInfoGetter, r.projectProvider, r.privilegedProjectProvider, r.settingsProvider)),
+		backupstoragelocation.DecodeGetBSLReq,
+		handler.EncodeJSON,
+		r.defaultServerOptions()...,
+	)
+}
+
+func (r Routing) deleteBackupStorageLocation() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+			middleware.SetClusterProvider(r.clusterProviderGetter, r.seedsGetter),
+			middleware.SetPrivilegedClusterProvider(r.clusterProviderGetter, r.seedsGetter))(backupstoragelocation.DeleteEndpoint(r.userInfoGetter, r.projectProvider, r.privilegedProjectProvider, r.settingsProvider)),
+		backupstoragelocation.DecodeDeleteBSLReq,
+		handler.EncodeJSON,
+		r.defaultServerOptions()...,
+	)
+}
```

Comment:
All of the routes are missing swagger documentation. As an example https://github.com/kubermatic/dashboard/blob/main/modules/api/pkg/handler/routes_v1_admin.go#L656-L667